src/Hangfire.Mongo/DistributedLock/MongoDistributedLock.cs

```diff
@@ -236,7 +202,7 @@ private void Cleanup()
             try
             {
                 // Delete expired locks
-                _database.DistributedLock.DeleteOne(
+                _database.DistributedLock.DeleteMany(
```

Comment:
Can't remember... I'll double check

Reply:
Well, looking at the code, it is in the "CleanUp" method. And even though there should only be one (if any) that matches the filter, I though better safe than sorry.
I do not think it will affect performance of functionality - but instead help out in situations that should never occur.

If you feel strongly about it, I'll change it back...